README.md

```diff
@@ -41,6 +41,12 @@ INSTALLED_APPS = [
 ]
 ```
 
+Run `collectstatic --link` to collect the necessary static files:
```

Comment:
@FrisoHarlaar collectstatic is part of deploying static files for any Django app (see https://docs.djangoproject.com/en/3.2/howto/static-files/#deployment)

Having said that, I agree that a note would help people.
How about: 
```suggestion
`wagtail-localize` loads additional assets for the editing interface. Run the `collectstatic` management command to collect all the required assets.
```

Reply:
@zerolab I'm still relatively new to Django and it took me quite a long time to figure out the problem, so I thought it would be nice to add to the note for other people. 

Your suggestion looks great!